server/main.lua

```diff
@@ -547,6 +547,65 @@ AddEventHandler('police:server:SendPoliceEmergencyAlert', function(streetLabel,
     --TriggerClientEvent("qb-phone:client:addPoliceAlert", -1, alertData)
 end) ]]
 
+RegisterServerEvent('police:server:checkBank')
+AddEventHandler('police:server:checkBank', function(playerId)
+    local src = source
+    local SearchedPlayer = QBCore.Functions.GetPlayer(playerId)
+    if SearchedPlayer ~= nil then 
+        TriggerClientEvent('chat:addMessage', source, {
+        template = '<div class="chat-message success">Player has €'..SearchedPlayer.PlayerData.money["bank"]..' in his bank account.</div>',
+        })
+    end
+end)
+
+RegisterServerEvent('police:server:checkLicenses')
+AddEventHandler('police:server:checkLicenses', function(playerId)
+    local src = source
+    local SearchedPlayer = QBCore.Functions.GetPlayer(playerId)
+    if SearchedPlayer ~= nil then 
+        local licences = SearchedPlayer["PlayerData"]['metadata']['licences']
+        local str = ""
+        local index = 0
+        local max = 0
+
+        for k,v in pairs(licences) do
+            if k and v then
+                str = str .. k:gsub("^%l", string.upper) .. ', '
+            end
+        end
+        TriggerClientEvent('chat:addMessage', source, {
+            template = '<div class="chat-message success">Player Licenses: ' .. (#str == 0 and "No Licenses" or str:sub(1,#str-2)) .. '</div>',
+        })
+    end
+end)
+
+RegisterServerEvent('police:server:checkFines')
+AddEventHandler('police:server:checkFines', function(playerId)
+    local src = source
+    local SearchedPlayer = QBCore.Functions.GetPlayer(playerId)
+    if SearchedPlayer ~= nil then 
+        QBCore.Functions.ExecuteSql(false, "SELECT * FROM phone_invoices WHERE `citizenid` = '"..SearchedPlayer.PlayerData.citizenid.."' AND `society`='police'", function(invoices)
```

Comment:
please use exports.ghmattimysql:executeSync because qb-core already remove this function